src/pages/Settings.tsx

```
@@ -326,7 +342,7 @@ function renderSetting(
             value={getSetting(setting.id).toString()}
             onValueChange={(value: string) => {
               if (value !== undefined) {
-                setSetting(setting.id, value);
+                setSetting(setting.id as any, value);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)